one_fm/public/js/desk.js

```diff
@@ -42,43 +42,8 @@ let improve_my_erp = () => {
 	let improveBTN = document.createElement('a');
 	improveBTN.classList = "btn btn-default btn-xs improve-my-erp";
 	improveBTN.textContent = "Improve";
+  improveBTN.href = '/helpdesk/tickets/new'
```

Comment:
This looks wrongly indented, please is this correct ?

Reply:
This href is for redirection to Helpdesk. I am sorry I am not sure what you are actually asking